diff -u -r ladspa_sdk/src/makefile ladspa_sdk/src/makefile
--- ladspa_sdk/src/makefile	2007-11-06 18:42:45.000000000 +0800
+++ ladspa_sdk/src/makefile	2023-04-17 23:24:34.000000000 +0800
@@ -25,8 +25,8 @@
 PROGRAMS	=	../bin/analyseplugin				\
 			../bin/applyplugin 				\
 			../bin/listplugins
-CC		=	cc
-CPP		=	c++
+CC		?=	cc
+CPP		?=	c++
 
 ###############################################################################
 #
